gen.pollinations.ai/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+    "compilerOptions": {
+        "target": "ES2022",
+        "module": "ESNext",
+        "moduleResolution": "bundler",
+        "lib": ["ES2022"],
+        "types": ["@cloudflare/workers-types"],
```

Comment:
Just as a note: as far as i know using`@cloudflare/workers-types` is the old way of doing it, we get much nicer types by running `wrangler types` and using the generated `worker-configurations.d.ts`.